vehicles/src/modules/permit-application-payment/permit/permit.controller.ts

```diff
@@ -170,10 +171,10 @@ export class PermitController {
     description:
       'Sends a notification related to a specific permit after checking user authorization.',
   })
-  @Roles({
-    userAuthGroup: IDIR_USER_AUTH_GROUP_LIST,
-    oneOf: [Role.SEND_NOTIFICATION],
-  })
+  @Permissions({
+    userRole: IDIR_USER_ROLE_LIST,
+    oneOf: [Claim.SEND_NOTIFICATION],
+  } as IRole)
```

Comment:
I dont think you need to typecast it as IRole

Reply:
Same comment as above.